### ISSUE TITLE
The string must not be provided only by red color

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1944]
URL: http://clarity-ks.surge.sh/Tables-bars

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Tables tab.
3. Use WAT (Colour - Greyscale) where content may be color dependent to determine if the meaning is clear when the color is removed in Tables tab.

Actual Results:
Color is the only means to convey information “td” for Left-aligned table cells in Tables tab.
For more details, please refer to "Tables-InfomationIsProvidedByColor.png".

Expected Results:
Color cannot be the only means to convey information “td”.
Information conveyed through color must also be provided in text on the screen.

Fix Suggestion:
The same information is represented through other visual (non-color) differentiation, such as Shape, Position or Size.

Note:
This issue also occurs on “Selectors” column for “Headers” and “Body text and paragraphs” data table in Typography tab.

Violation of Section 508 Standards: 
Section 1194.22(c) Web pages shall be designed so that all information conveyed with color is also available without color, for example from context or markup.

User Impact: Blind and color blind.

Screen shot:
1.	Tables-InfomationIsProvidedByColor.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/cc34a0e6-4aed-4e4b-86ab-727f8eeb5470)

NOTE: I need convincing on this one...

**Comment by mathisscott** [comment]
This is just the code highlighting. There is no meaningful difference from surrounding content. It is purely decorative.
